charts/home-assistant/README.md

```diff
@@ -57,8 +57,8 @@ The following tables lists the configurable parameters of the Home Assistant cha
 | `probes.startup.failureThreshold`               | Specify startup `failureThreshold` parameter for the deployment                                                                                                                                                                           | `5`                                    |
 | `probes.startup.periodSeconds`                  | Specify startup `periodSeconds` parameter for the deployment                                                                                                                                                                              | `10`                                   |
 | `service.type`                                  | Kubernetes service type for the home-assistant GUI                                                                                                                                                                                        | `ClusterIP`                            |
-| `service.port`                                  | Kubernetes port where the home-assistant GUI is exposed                                                                                                                                                                                   | `8123`                                 |
-| `service.portName`                              | Kubernetes port name where the home-assistant GUI is exposed                                                                                                                                                                              | `api`                                  |
+| `service.port`                                  | Port to be exposed by the home-assistant GUI service                                                                                                                                                                                      | `8123`                                 |
+| `service.portName`                              | Name of the port exposed within the home-assistant GUI service                                                                                                                                                                            | `api`                                  |
```

Comment:
This field is unnecessary and could be removed but would likely be a breaking change?

Reply:
Yes, in this case you'd want to bump the version to 3.0.0

Reply:
If you are removing the config for ports, you can remove them from the readme too.

Reply:
and make a note in the readme on the breaking changes, e.g. https://github.com/k8s-at-home/charts/tree/master/charts/blocky#upgrading-an-existing-release-to-a-new-major-version

Reply:
@onedr0p would you be ok with shipping this as is and I'll open an issue to track removing it? Reason being I'm planning to dig in deep this weekend on Home Assistant and may find some other potentially breaking changes.

Reply:
I'm a bit confused, you removed the variables in the templates but kept them in the readme?

Reply:
Ah sorry for not being clear. I didn't remove the `service.portName` variable. I only cleaned up the description here. I think there may have been a misunderstanding when this chart was created that this port name needed to match the name in the deployment.

Reply:
I think this is fine to merge, you are right that these apps don't have a current way to support changing the ports the apps are running on within the container.

Reply:
or if they do, we don't give a way for people to change it.